src/pineko/cli/check.py

```diff
@@ -72,18 +62,41 @@ def sub_scvar(grid_path, scale, max_as_order, max_al_order):
     grid = pineappl.grid.Grid.read(grid_path)
     grid.optimize()
     success = "[green]Success:[/] grids contain"
+    warning = "[orange]Warning:[/] grids do not contain central order for requested"
     error = "[red]Error:[/] grids do not contain"
     # Call the function
-    try:
-        conditions = Scale[scale].value.check(grid, max_as_order, max_al_order)
-    except KeyError:
-        raise ValueError("Scale variation to check can be one between ren and fact")
-    for coupling, condition in zip(Coupling, conditions):
+    scaleobj = check.Scale[scale]
+    checkres, max_as_effective = check.contains_sv(
+        grid, max_as_order, max_al_order, scaleobj
+    )
+    sv_as = None
+    central_as = None
+    if max_as_effective == max_as_order:
+        if checkres is check.AvailableAtMax.BOTH:
+            sv_as = True
+            central_as = True
+        elif checkres is check.AvailableAtMax.CENTRAL:
+            sv_as = False
+            central_as = True
+        else:
+            sv_as = True
+            central_as = False
+    else:
+        sv_as = False
+        central_as = False
+
+    sv_conditions = [sv_as, True]
+    cen_conditions = [central_as, True]
+    for coupling, sv_condition, cen_condition in zip(
+        Coupling, sv_conditions, cen_conditions
+    ):
         to_write = ""
-        if condition:
+        if sv_condition and cen_condition:
             to_write += success
+        elif sv_condition and not cen_condition:
+            to_write += warning
         else:
             to_write += error
-        to_write += " " + Scale[scale].value.descr
+        to_write += " " + check.Scale[scale].value.description
         to_write += f" for {coupling.name.lower()}"
```

Comment:
Instead of breaking the message in many pieces, and adding them together (that makes the overall message unclear) just use a unique template.
Something like:
```py
message = messages[condition]
descr = check.Scale[scale].value.description
cname = coupling.name.lower()
rich.print(f"{message} {descr} for {cname}")
```